src/app/_components/Meta.tsx

```diff
@@ -14,7 +14,7 @@ export function Meta({ metaData }: MetaProps) {
   }
 
   return (
-    <div className="flex gap-3 text-brand-300">
+    <div className="flex flex-wrap gap-2 text-brand-300">
```

Comment:
```suggestion
    <div className="flex flex-wrap gap-x-2 gap-y-1 text-brand-300">
```

Reply:
Would that solve the problem you mentioned @barbaraperic?

Reply:
Nice 😍